packages/template-typescript-minimal/app/components/_app/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {Provider} from 'slas-react-sdk'
+import {getAppOrigin} from 'pwa-kit-react-sdk/utils/url'
+
+interface AppProps {
+    children?: React.ReactNode
+}
+
+const App = ({children}: AppProps) => {
+    return (
+        <Provider
+            proxy={`${getAppOrigin()}/mobify/proxy/api`}
+            organizationId="f_ecom_zzrf_001"
+            clientId="c9c45bfd-0ed3-4aa2-9971-40f88962b836"
+            siteId="RefArch"
+            shortCode="8o7m175y"
```

Comment:
I think this type is ShopperLoginParameters... open question on whether or not it's a good idea to make it a single prop called config. E.g. I don't think you would ever use `clientId` and not `siteId`, therefore they have a logical grouping.